shared/cfg/machines.cfg

```diff
@@ -45,10 +45,8 @@ variants:
         # TODO: Change to "gic-version=max" when supported
         machine_type_extra_params = gic-version=host
         # No support for VGA yet
-        vga = none
+        vga = virtio
```

Comment:
The above comment should be modified as well.

Reply:
yep